sql/moz-fx-data-shared-prod/apple_ads_external/ios_app_campaign_stats_v1/metadata.yaml

```diff
@@ -4,14 +4,19 @@ description: |-
 owners:
 - kwindau@mozilla.com
 labels:
-  incremental: false
+  incremental: true
   owner1: kwindau@mozilla.com
 scheduling:
   dag_name: bqetl_ios_campaign_reporting
   depends_on_past: false
-  date_partition_parameter: null
 bigquery:
-  time_partitioning: null
+  time_partitioning:
+    type: day
+    field: date
+    require_partition_filter: false
   clustering:
-    fields: [campaign]
-references: {}
+    fields:
+    - campaign
```

Comment:
@kwindau how many entries do we expect in this table for each campaign? Does it make sense to cluster here?